src/GraphQL.DataLoader.Tests/QueryTestBase.cs

```diff
@@ -18,7 +18,7 @@ namespace GraphQL.DataLoader.Tests
 {
     public abstract class QueryTestBase : DataLoaderTestBase
     {
-        private readonly IDocumentExecuter executer = new DocumentExecuter();
+        private readonly IDocumentExecuter executer = new SubscriptionDocumentExecuter();
```

Comment:
Note that in most tests it was not necessary to make this replacement.

Reply:
That executer is a test class, right?  It's not part of GraphQL proper.

Reply:
`SubscriptionDocumentExecuter` ? It is 'real` class from GraphQL.Subscriptions project. DataLoader.Tests project calls `var result = await ExecuteSubscribeAsync("subscription OrderAdded { orderAdded { orderId } }");` so we need `SubscriptionDocumentExecuter`.

Reply:
Can you point me to the project?  I am not familiar with it.

Reply:
https://github.com/graphql-dotnet/graphql-dotnet/pull/1973/files#r514568778

This is **new** project :)

Reply:
Duh.  Obviously I didn't read the PR.... lol

Reply:
Everything turned out pretty simple.